src/commonMain/kotlin/fr/acinq/lightning/payment/LiquidityPolicy.kt

```diff
@@ -3,38 +3,47 @@ package fr.acinq.lightning.payment
 import fr.acinq.bitcoin.Satoshi
 import fr.acinq.lightning.LiquidityEvents
 import fr.acinq.lightning.MilliSatoshi
-import fr.acinq.lightning.logging.*
+import fr.acinq.lightning.logging.MDCLogger
 import fr.acinq.lightning.utils.msat
+import fr.acinq.lightning.utils.sat
 import fr.acinq.lightning.utils.toMilliSatoshi
 
-
 sealed class LiquidityPolicy {
-    /** Never initiates swap-ins, never accept pay-to-open */
+    /** Never initiates swap-ins, never accept on-the-fly funding requests. */
     data object Disable : LiquidityPolicy()
 
     /**
-     * Allow automated liquidity managements, within relative and absolute fee limits. Both conditions must be met.
+     * Allow automated liquidity management, within relative and absolute fee limits. Both conditions must be met.
+     *
+     * @param inboundLiquidityTarget amount of inbound liquidity the buyer would like to maintain (can be set to null to disable)
```

Comment:
That's a good point, initially I wanted to only request additional inbound liquidity if our current inbound was below our target. But for simplicity I currently implemented the following behavior:

- on new channel creation, we request `inboundLiquidityTarget`
- on splice-in, we don't request anything
- on on-the-fly funding, we request `inboundLiquidityTarget` (even if we already had that amount available)

Which of those do you think I should change?

Note that your proposed comment wouldn't be correct either, since right now we don't re-request inbound liquidity when doing a splice-in.

Reply:
> Which of those do you think I should change?

I think the current behavior is fine for now.

> Note that your proposed comment wouldn't be correct either, since right now we don't re-request inbound liquidity when doing a splice-in.

Right. The `LiquidityPolicy` probably needs to handle separately the on-chain and off-chain case, but this has high-level UX ramifications and can also be dealt with later.

Reply:
I slightly changed the wording in the latest rebase to be more vague :)